src/App.js

```diff
@@ -1,16 +1,44 @@
-import React from 'react';
+import {
+  BrowserRouter,
+  Routes,
+  Route,
+  Outlet,
+} from 'react-router-dom';
+
 import './App.css';
 import Calculator from './components/Calculator';
 import ApiNinja from './components/apicall';
+import Home from './components/home';
 
-function App() {
+function Layout() {
   return (
-    <div className="App">
-      <header className="App-header" />
-      <div><Calculator /></div>
-      <div><ApiNinja /></div>
+    <div className="body">
+      <div className="dflex spacebetween navcont">
+        <h1>Math Magicians</h1>
+        <div className="dflex mr10">
+          <h2 className="h2-class"><a href=".">Home</a></h2>
+          <h2 className="h2-class"><a href="./calculator">Calculator</a></h2>
+          <h2 className="h2-class"><a href="./quote">Quote</a></h2>
+        </div>
+      </div>
```

Comment:
- [Optional] To prevent the default browser behavior, it is advisable to use `Link` in react than the anchor tag.